x-pack/plugins/observability_onboarding/public/components/shared/optional_form_row.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiFormRowProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { euiThemeVars } from '@kbn/ui-theme';
+import React from 'react';
+
+type OptionalFormRowProps = EuiFormRowProps;
+
+export function OptionalFormRow(props: OptionalFormRowProps) {
+  const { label, children, helpText } = props;
+  return (
+    <EuiFormRow
+      css={{
+        '.euiFormLabel': {
+          width: '100%',
+        },
+        '.euiFormLabel > .euiFlexGroup > div:last-of-type': {
+          fontWeight: 'normal',
+          color: euiThemeVars.euiTextSubduedColor,
```

Comment:
We should probably use:
```typescript
import {  useEuiTheme } from '@elastic/eui';
...
const { euiTheme } = useEuiTheme();
```
and then set
```typescript
{ color: euiTheme.colors.subduedText }
```

Color values in `euiThemeVars` are not always the same as the EUI colors. In this case, dev tools shows the colors to be slightly off.